packages/integration/src/github/GithubCredentialsProvider.ts

```diff
@@ -233,7 +233,10 @@ export class GithubCredentialsProvider {
     const parsed = parseGitUrl(opts.url);
 
     const owner = parsed.owner || parsed.name;
-    const repo = parsed.owner ? parsed.name : undefined;
+    let repo = parsed.owner ? parsed.name : undefined;
+    // the github-discovery plugin passes an • as a repo name. This simply means it wants access to all repos
+    // that are available to the installation.
+    repo = repo === '*' ? undefined : repo;
```

Comment:
Can we skip this? 😅 I'm assuming it's referring to the GitHub discovery processor in the catalog? Thinking the URL would just be `https://github.com/backstage` for when you want to ingest all repos in an org. That or make it clear that this is a feature and document it above.

Reply:
I have moved the handling of this special case back into the GitHub discovery processor. Let me know how it looks. I'm not sure how confident I am about how I have chosen to build the GitHub Org URL.